pkg/policy/api/rule_validation.go

```diff
@@ -401,6 +414,20 @@ func (pp *PortProtocol) sanitize() (isZero bool, err error) {
 	return isZero, err
 }
 
+func (ir *ICMPRule) sanitize() error {
```

Comment:
nit: `sanitize` implies that `ir` is sanitized, i.e. modified. `verify` would probably be a better name.